src/drivers/rc/crsf_rc/CrsfRc.cpp

```diff
@@ -182,6 +177,47 @@ void CrsfRc::Run()
 	const hrt_abstime time_now_us = hrt_absolute_time();
 	perf_count_interval(_cycle_interval_perf, time_now_us);
 
+	if (_vehicle_status_sub.updated()) {
+		vehicle_status_s vehicle_status;
+
+		if (_vehicle_status_sub.copy(&vehicle_status)) {
+			_armed = (vehicle_status.arming_state == vehicle_status_s::ARMING_STATE_ARMED);
+		}
+	}
+
+	/* vehicle command */
+	vehicle_command_s vcmd;
+
+	if (_vehicle_cmd_sub.update(&vcmd)) {
+		// Check for a pairing command
+		if (vcmd.command == vehicle_command_s::VEHICLE_CMD_START_RX_PAIR) {
+
+			uint8_t cmd_ret = vehicle_command_ack_s::VEHICLE_CMD_RESULT_UNSUPPORTED;
+
+			if (!_is_singlewire && !_armed) {
+				if ((int)vcmd.param1 == 1) {
```

Comment:
We should change this to not be a magic number. I'll discuss the MAVLink PR this evening.

Reply:
I agree, if the mavlink PR goes in, will the enum be available here?

Reply:
I merged your mavlink PR (thanks!). 

I believe that this mavlink PR will automatically get updated with the changes in the mavlink/mavlink submodule, and when it gets merged that will include the update https://github.com/PX4/PX4-Autopilot/pull/23495

@dagar That's how it works right?

Reply:
@hamishwillee I was able to use the updated mavlink enum in QGC, however it doesn't appear available in PX4.
https://github.com/mavlink/qgroundcontrol/pull/12068/commits/2a8bcdf85a732c62c74d5c250c229fa1ed9a8885

I've seen duplicate enum definitions copied from the common.xml into header files for different modules. Should I do the same here, or is there a way to map to the mavlink common.xml?

Reply:
@dagar I've updated vehicle_command with the mavlink enums. Let me know how this looks